server/agents/agents.py

```diff
@@ -1,3 +1,4 @@
+"""Base class for all agents."""
```

Comment:
file-level comments should be below the copyright header

Reply:
will do -- Duckie led me astray